Cargo.lock

```diff
@@ -332,7 +332,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "process_viewer"
-version = "0.0.16"
+version = "0.0.17"
```

Comment:
You're not supposed to change this file though. If you want me to publish a new version, just update the `Cargo.toml` file.

Reply:
Ok, will fix this. I didn't do this change manually, some script did it, maybe a git hook?

Reply:
Done.